lms/static/scripts/frontend_apps/components/FileList.js

```diff
@@ -39,14 +39,13 @@ export default function FileList({
         renderItem={(file, isSelected) => (
           <Fragment>
             <td aria-label={file.display_name} className="FileList__name-col">
-              {/* FIXME-A11Y */}
-              {/* eslint-disable-next-line jsx-a11y/alt-text */}
               <img
                 className={classnames(
                   'FileList__icon',
                   isSelected && 'is-selected'
                 )}
                 src="/static/images/file-pdf.svg"
+                alt="File icon"
```

Comment:
Perhaps this should be more specific and say "PDF icon". On the other hand the most likely future change that I can think of is that we add folders to this list in which case the most important differentiation will be between files and folders.